app/index.html

```diff
@@ -16,6 +16,7 @@
 <link rel="icon" sizes="512x512" href="img/icons/512.png">
 <meta name="application-name" content="Foxbox Client">
 <title>Foxbox Client</title>
+<script type="text/javascript" src="cordova.js"></script>
```

Comment:
Can you remove the `type="text/javascript"` attribute and add a `defer` one, for the sake of consistency?